lib/idp/constants.rb

```diff
@@ -70,6 +70,7 @@ module Vendors
       GU
       MP
       PR
+      UM
```

Comment:
This is "outlying virgin islands". I feel like maybe this doesn't belong in this PR?

Reply:
I'm glad you caught this omission!  I wasn't aware of it.

Reply:
Yes, thank you for catching it!

I am pondering if it should be a different PR. I think that would give some clarity to the issue, though if this is the only place it's been missed, maybe that clarity isn't worth the effort involved in a new PR?

I am wondering if there are other places where it might be needed, like [here](https://github.com/18F/identity-idp/blob/main/config/application.yml.default#L92) and a few other places in `application.yml.default`?

I searched for every place `GU` was used, hoping it might be a map to other places it could have been omitted. Though I am unsure how to tell if`UM` was left out unintentionally or intentionally in these types of variables.

Reply:
I'm going to remove this from this PR. There's another related discussion [here](https://gsa-tts.slack.com/archives/C056RD1NEHW/p1701456751547049).